.gitignore

```diff
@@ -38,3 +38,5 @@ yarn.lock
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+
+.node-version
```

Comment:
You may want to consider adding this to your global `.gitignore`, too (or instead?)

Reply:
Good point, I just removed it.